src/components/ResultWrapper/ResultWrapper.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import AnalysisWrapper from '../AnalysisWrapper/AnalysisWrapper';
+
+const ResultWrapper = (props) => {
+  const { response, loadingAnalysis, event } = props;
+
+  const renderAction = (action) => {
+    /* if (action === 'anonymize' && response.anonymizeResult) {
```

Comment:
Would be cleaner if this comment where removed

Reply:
will include a anonymize function soon, so this comment will be used, but ok